book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -29,17 +29,33 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 - [Support additional users](#support-additional-users)
 - [Requires additional skills](#requires-additional-skills)
 
-## Plead the fifth
+## Limited incentives to give evidence against yourself
 
 The Fifth Amendment to the United States Constitution includes a clause that no one "shall be compelled in any criminal case to be a witness against [them]sel[ves]" [REF](https://en.wikipedia.org/wiki/Fifth_Amendment_to_the_United_States_Constitution).
```

Comment:
How about [Wikipedia.org - Fifth Amendment] for the reference? As it's an external link we tend to just link directly

Reply:
Fixed in 8193d1b